runtime/doc/usr_27.txt

```diff
@@ -83,7 +83,7 @@ matter if 'ignorecase' or 'smartcase' was changed.
 
 	Note:
 	If your search takes much longer than you expected, you can interrupt
-	it with CTRL-C on Unix and  CTRL-Break on MS-DOS and MS-Windows.
+	it with CTRL-C on Unix and CTRL-Break on Windows.
```

Comment:
I'll try with Vim. Neovim doesn't work in the terminal on Windows so can't test that currently.

What's a good way to test this?

```
:sleep <value>
<CTRL-C>
```

I'll admit my ignorance about never using the sleep feature. :)

Above seems to work on Neovim on Unix though...